### DISCUSSION TITLE
GCBM Internationalization (Multiple Languages)

### DISCUSSION THREAD

**Discussion by laetemn** [AxionDrak/GameCube-Backup-Manager#92] [Ideas]
Searching the Internet about software internationalization, I found something very interesting.

Basically, it's the GNU gettext system ported to C#.

The gettext system is widely used in software with multiple languages ​​due to the ease of creating and translating Language packs (thanks to the .mo and .po files).

Apparently the "Wii Backup Manager" program uses a similar mechanism through Plain Text files (key = value).

Perhaps using the GetText system can speed up the availability of new languages for GCBM.

Here's an open discussion on the subject...

Link 1: https://www.gnu.org/software/gettext/

Link 2: https://www.gnu.org/software/gettext/manual/html_node/C_0023.html

Link 3: https://github.com/perpetualKid/GetText.NET

Link 4: https://hudecekpetr.cz/multilanguage-game-using-gnu-gettext-in-c/

**Comment by sjohnson1021** [comment]
In the meantime, I've exported the latest version of the resources files in an excel format and hosted it on Google sheets.

I've posted this link to the thread over at GBATemp as well.

This should facilitate much easier access, and open up translation to the wider public. Two heads are better than.. well you get the idea. 

Once again, any and all assistance is appreciated.

> ## [Google Sheet - GCBM Localization Project ](https://docs.google.com/spreadsheets/d/1YyImjM7hPkJPdr_8nmBuiRnnrPzdmb2J/edit?usp=drivesdk&ouid=113862669718221596237&rtpof=true&sd=true)
